<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchAdd</class>
 <widget class="QDialog" name="BatchAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>215</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>批量添加数据项</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/new/prefix/img/Cart_add_128px_1198145_easyicon.net.ico</normaloff>:/new/prefix/img/Cart_add_128px_1198145_easyicon.net.ico</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>数据个数：</string>
   </property>
   <property name="buddy">
    <cstring>lineEditNum</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNum">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>数据个数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>起始数据：</string>
   </property>
   <property name="buddy">
    <cstring>lineEditStartData</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditStartData">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>起始数据</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>数据步长：</string>
   </property>
   <property name="buddy">
    <cstring>lineEditDataStep</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDataStep">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>数据步长</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>起始地址：</string>
   </property>
   <property name="buddy">
    <cstring>lineEditStartAddr</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditStartAddr">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>十进制或十六进制</string>
   </property>
   <property name="placeholderText">
    <string>起始地址</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtnGenerate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="whatsThis">
    <string>按条件生成数据项</string>
   </property>
   <property name="text">
    <string>生成</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtnReset">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="whatsThis">
    <string>清空所有参数</string>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>数据增长
        方向：</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioBtnRise">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>升序递增</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioBtnDrop">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>降序递减</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
